README.md

```diff
@@ -16,6 +16,7 @@ Apiary consists of the following components which are managed in separate git re
 * [Apiary Metastore Docker](https://github.com/ExpediaInc/apiary-metastore-docker)
 * [Apiary Federation](https://github.com/ExpediaInc/apiary-federation)
 * [Apiary Waggle Dance Docker](https://github.com/ExpediaInc/apiary-waggledance-docker)
+* [Apiary Authorization](https://github.com/ExpediaInc/apiary-authorization)
 
```

Comment:
Could you also add the link to the Ranger docker repo - https://github.com/ExpediaInc/apiary-ranger-docker?

Reply:
done